include/config/overworld.h

```diff
@@ -51,7 +51,7 @@
                                                   // (You should not use 48x48 sprites/tables for compressed gfx)
                                                   // 16x32, 32x32, 64x64 etc are fine
 // Follower Pokémon
-#define OW_FOLLOWERS_ENABLED           FALSE      // Enables follower Pokémon, HGSS style. Requires OW_POKEMON_OBJECT_EVENTS. Note that additional scripting may be required for them to be fully supported!
+#define OW_FOLLOWERS_ENABLED           FALSE // Enables follower Pokémon, HGSS style. Requires OW_POKEMON_OBJECT_EVENTS. Note that additional scripting may be required for them to be fully supported!
```

Comment:
This needs to be reverted

Reply:
[`6bcd9f8` (#5289)](https://github.com/rh-hideout/pokeemerald-expansion/pull/5289/commits/6bcd9f87f055f482386b5791bb136aaf948915f0)